lms/templates/main.html

```diff
@@ -175,6 +175,8 @@
   % endif
 
   <%block name="js_extra"/>
+
+  <script type="text/javascript" src="${settings.STATIC_URL}js/utils/dropdown.js"></script>
```

Comment:
This may no longer be necessary. I've added it to the common.py payload. I'll investigate.